src/main/java/com/shaft/gui/element/ElementActions.java

```diff
@@ -533,7 +533,15 @@ public static void switchToIframe(WebDriver driver, By elementLocator) {
     public static void switchToWindow(WebDriver driver, String nameOrHandle) {
         WebDriverElementActions.switchToWindow(driver, nameOrHandle);
     }
-
+    /**
+     * Switches focus to another Tap
+     *
+     * @param driver       the current instance of Selenium webdriver
+     * @param URL The name of the URL you want to navigate to
+     */
+    public static void switchToNewTap(WebDriver driver, String URL) {
```

Comment:
opening a link in a new tab is not an element action.
kindly move it to browser actions instead.
also please note that every method has two variants, one is static and one is a regular method. the static method takes a driver object as a parameter and the regular method uses the existing driver object.

Reply:
Done (Y)